File: Codeforces/#E97_DIV2/!A.cpp
#include <iostream>
#include <iomanip>

using namespace std;
int t,l,r;

int solution(long long l, long long r){
    bool is_success = false;
    long long int a;

    a = l * 2 / 3;
    if (a * 2 > r) {
        is_success = true;
    }
    // cout << a << endl;

    a = l * 2;
    if (a > r) {
        is_success = true;
    }
    // cout << a << endl;

    if (is_success){
        cout << "YES" << endl;
    } else {   
        cout << "NO" << endl;
    }

    return 0;  
}


int main() {
    // freopen("input.txt","r",stdin);
    int ptime = clock();

    cin >> t;
    for (int i=0;i<t;i++){
        cin >> l >> r;
        solution(l,r);
    }
    
    // cout << "time : " << clock() - ptime << endl;

    return 0;
}

File: Codeforces/#711_DIV2/!A.py
import sys
import time

ptime = time.time_ns()

sys.stdin = open('input.txt', 'r')


#####################################################


def gcd(a, b):
    if b == 0:
        return a
    return gcd(b, a % b)

def solution(n):
    nsum = 0
    while n:
        nsum += n % 10
        n = n // 10
    print(nsum)
    print(gcd(n,nsum))
    return

t = int(input())
for tt in range(t):
    n = int(input())
    print(solution(n))



#####################################################



print('time', (time.time_ns() - ptime)/10**6,'ms')














